docs/developerguide.rst

```diff
@@ -9,4 +9,44 @@ understand REANA source code and contribute to it.
 Debugging
 ---------
 
-We use ``wdb``.
+Assuming that REANA is being run in development mode following
+`reana-cluster documentation
+<http://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions>`_,
+to debug a component we need to rebuild its Docker image in debug mode. Let say
+we are going to debug `reana-server`:
+
+.. code-block:: console
+
+   $ # if we are not connected to the minikube docker daemon we should do it
+   $ eval "$(minikube docker-env)"
+   $ docker build . --build-arg DEBUG=true -t reanahub/reana-server:v0.1.0
+
+Set the breakpoint in your current project code using your preferred editor and
+add `import wdb; wdb.set_trace()` instruction, for instance in the
```

Comment:
I would suggest moving this example to a separate block like you did in https://github.com/reanahub/reana-resources-k8s/pull/28#issuecomment-334534629, LGTM in general

Reply:
I did this by this time because you had to edit the file inside the container:
```console
$ kubectl exec -ti [pod_name] bash
> emacs|vim /path/to/file/where/to/set/breakpoint.py
# Inside the file set the breakpoint as follows:
# import wdb; wdb.set_trace()
# save the file. Since we have FLASK_DEBUG active Flask will reload.
```
However, now it is possible to do it from your editor thanks to the possibility of mounting your code inside the containers. Because of that I didn't include a code block. What do you think about adding an screenshot like this?
<img width="544" alt="screen shot 2018-02-14 at 3 24 31 pm" src="https://user-images.githubusercontent.com/9035606/36209152-4157b1ce-119b-11e8-837c-b3f93ecaeef6.png">

Reply:
Yes, quite illustrative!

Reply:
Adding illustrative screenshot is nice.  You can join it with the using-detailed-walk-through example comment of mine, and use a tone such as:

"Let us introduce the ``wdb`` breakpoint as the first instruction of the ``get_analyses()`` function located in ``reana_server/rest/analyses.py``:"